parsers/cxx/cxx_parser_tokenizer.c

```diff
@@ -1088,16 +1140,30 @@ boolean cxxParserParseNextToken(void)
 				))
 			{
 				CXX_DEBUG_PRINT("Ignore token %s",vStringValue(t->pszWord));
-				// FIXME: Handle ignore parens!
+
 				if(szReplacement && *szReplacement)
 				{
+					CXX_DEBUG_PRINT("The token has replacement %s: applying",szReplacement);
 					vStringClear(t->pszWord);
 					vStringCatS(t->pszWord,szReplacement);
+					// FIXME: Do we need to interpret the replacement?
```

Comment:
I can easily handle single keywords. Combinations are harder to do since they would require the token to be splitted again with a custom algorithm, then one would need to keep a backlog of tokens yet to be returned which would affect the performance of the existing algo... I don't think it's worth the overhead. I'll add a check for keywords.